extensions/amp-analytics/0.1/storage-impl.js

```diff
@@ -302,13 +305,21 @@ export class LocalStorageBinding {
   /** @override */
   loadBlob(origin) {
     return new Promise(resolve => {
+      if (!this.isLocalStorageSupported_) {
+        resolve();
```

Comment:
Since `loadBlob` is defined as a promise resolving null or string - let's do `resolve(null);`